### ISSUE TITLE
not compiled with crystal master

### ISSUE THREAD

**Issue by kostya** [stefanwille/crystal-redis#30]
`Crystal 0.18.7+357 [07964d0] (2016-08-23)`

```
crystal spec
Error in line 1: while requiring "./spec/redis_spec.cr"

in ./spec/redis_spec.cr:1: while requiring "./spec_helper"

require "./spec_helper"
^

in ./spec/spec_helper.cr:2: while requiring "../src/redis"

require "../src/redis"
^

Syntax error in ./src/redis.cr:95: unexpected token: ( (expecting when, else or end)

      select(database)
            ^

```

i guess `select` is now language keyword

**Comment by stefanwille** [comment]
Thanks for reporting.

**Comment by stefanwille** [comment]
Fixed in v1.6.7
